prototype_v1_JC.yml

```diff
@@ -3,26 +3,37 @@ author:
     string-name:
     surname: 
     given-names: 
-    email: 
+    email:
+    phone:
     PID:
       type: orcid-id
       id: 
     role:
+      vocab-term-identifier: free text explanation
+      vocab-term-identifier: free text explanation
     affiliation:
       aff_01
     part-of-group: NA   
     attributes:
-      is.corresponding.author: no
+      is-corresponding-author: no
       type: author
+      deceased: no
+      equal contribution: no
```

Comment:
Should this be `equal-contribution`?

Reply:
```suggestion
      equal-contrib: no
```

Reply:
the jats entry is equal-contrib probably good to keep the same one, thanks @the-solipsist to have spotted this!